python_bindings/tutorial/lesson_11_cross_compilation.py

```diff
@@ -93,7 +96,7 @@ def main():
                                 1]       # Current version of elf
 
         length = len(arm_32_android_magic)
-        f = open("lesson_11_arm_32_android.o", "rb")
+        f = open(os.path.join(tmpdir, "lesson_11_arm_32_android.o"), "rb")
```

Comment:
If tmpdir were a `Path` variable (from the standard `pathlib` module) then you could just write: `tmpdir / "lesson_11_arm_32_android.o"`

Reply:
there is zero other usage of that in our current python; if we want to convert, let's do it all at once, not piecewise.